core/iwasm/libraries/thread-mgr/thread_manager.c

```diff
@@ -393,6 +393,11 @@ wasm_cluster_spawn_exec_env(WASMExecEnv *exec_env)
     wasm_runtime_set_custom_data_internal(
         new_module_inst, wasm_runtime_get_custom_data(module_inst));
 
+#if WASM_ENABLE_LIBC_WASI != 0
+    WASIContext* wasi_ctx = wasm_runtime_get_wasi_ctx(module_inst);
```

Comment:
Could you please define variable wasi_ctx at the beginning of the function, e.g. after line 365? Here it is C source code, we normally define variables at the beginning of function to avoid compile warnings reported by some compilers.